azblob/zt_policy_request_id_test.go

```diff
@@ -0,0 +1,101 @@
+package azblob
+
+import (
+	"context"
+	"errors"
+	"github.com/Azure/azure-pipeline-go/pipeline"
+	chk "gopkg.in/check.v1"
+	"net/http"
+	"net/url"
+)
+
+type requestIDTestScenario int
+
+const (
+	// Testing scenarios for echoing Client Request ID
+	clientRequestIDMissing requestIDTestScenario = 1
+	clientRequestIDError   requestIDTestScenario = 2
+	clientRequestIDMatch   requestIDTestScenario = 3
+	clientRequestIDNoMatch requestIDTestScenario = 4
+)
+
+type dummyPolicy struct {
```

Comment:
Minor: to future-proof this struct, consider renaming it with key word "client request ID" to clarify that its intention is to mimic different behavior related to the client request ID.

Ex: if we add another dummy policy for a different purpose in the future, someone may have to come back here to rename it.